client/python/modeldb/__main__.py

```diff
@@ -0,0 +1,182 @@
+import os
+import json
+
+
+def to_bool(value):
+    valid = {'true': True, 't': True, '1': True,
+             'false': False, 'f': False, '0': False,
+             }
+
+    if isinstance(value, bool):
+        return value
+
+    if not isinstance(value, basestring):
+        raise ValueError('invalid literal for boolean. Not a string.')
+
+    lower_value = value.lower()
+    if lower_value in valid:
+        return valid[lower_value]
+    else:
+        raise ValueError('invalid literal for boolean: "%s"' % value)
+
+
+if __name__ == '__main__':
+    execution_dir = os.getcwd()
+    file_dir = os.path.dirname(__file__)
+    filename = file_dir + '/syncer.json'
+    config = json.load(file(filename, 'r'))
+    new_config = json.load(file(filename, 'r'))
+
+    prompt = '> '
+
+    raw_input(
+        'This utility helps you create a modeldb configuration file.\n'
+        'The file is used by the modeldb python and/or scala clients to '
+        'connect to the modeldb server.\n\n'
+        'Press <ENTER> to use defaults')
+
+    print('\n== Output Filename ==')
+    prompt = 'output filename (default=custom_syncer.json): '
+    output_filename = str(raw_input(prompt))
+    if output_filename == '':
+        output_filename = 'custom_syncer.json'
+
+    print('\n== Thrift ==')
+
+    prompt = 'host (default=%s): ' % config['thrift']['host']
+    thrift_host = raw_input(prompt)
+    if thrift_host == '':
+        thrift_host = config['thrift']['host']
+    new_config['thrift']['host'] = thrift_host
+
+    prompt = 'port (default=%s): ' % (config['thrift']['port'])
+    thrift_port = raw_input(prompt)
+    if thrift_port == '':
+        thrift_port = config['thrift']['port']
+    thrift_port = int(thrift_port)
+    new_config['thrift']['port'] = thrift_port
+
+    print('\n== Syncing Strategy =='
```

Comment:
Some of these are not going to mean much to the user, so perhaps we just expose the relevant ones? Here are the ones that are relevant:

== Output Filename ==
output filename (default=custom_syncer.json): 

== Thrift ==
host (default=localhost): 
port (default=6543): 

== Syncing Strategy ==eager/lazy
syncing strategy (default=eager): 

== Project ==
name (default=Demo): 
author (default=modeldb_users): 
description (default=Project to hold all models from the demo): 

== Experiment ==
name (default=Default Experiment): 
description (default=Default Experiment): 

== Experiment Run Description ==
experiment run description (default=Default Run Description):